COPY/usr/lib/systemd/system/evminit.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description="Initializes the evm environment"
+After=syslog.target
+
+[Service]
+ExecStart=/bin/sh -c "/bin/evminit.sh"
+Type=oneshot
```

Comment:
Let's open an issue to make sure this is a "every boot" service. The `pvscan` needs to run every time the appliance boots (https://github.com/abellotti/manageiq-appliance/blob/centos71_dev/LINK/bin/evminit.sh#L19)